library/lib/application.cpp

```diff
@@ -229,23 +229,126 @@ bool Application::init(std::string title, Style* style, LibraryViewsThemeVariant
 #ifdef __SWITCH__
     {
         PlFontData font;
+        int locale = i18n::swGetCurrentLocaleID();
 
         // Standard font
         Result rc = plGetSharedFontByType(&font, PlSharedFontType_Standard);
         if (R_SUCCEEDED(rc))
         {
-            Logger::info("Using Switch shared font");
-            Application::fontStash.regular = Application::loadFontFromMemory("regular", font.address, font.size, false);
+            Logger::info("Adding Switch shared standard font");
+            Application::fontStash.standard = Application::loadFontFromMemory("regular", font.address, font.size, false);
```

Comment:
Wouldn't that be `standard` as well in the `loadFontFromMemory` call?

Reply:
changed